content/tracking/tracking_7.md

```diff
@@ -1,70 +1,208 @@
 
+#### `trackRefund`
+
+To track a refund you can use the `trackRefund` method with the following attributes:
+
 ```ts
-import { setPageType } from "@snowplow/browser-plugin-snowplow-ecommerce";
+import { trackRefund } from "@snowplow/browser-plugin-snowplow-ecommerce";
+
+trackRefund({
+  transaction_id,
+  currency,
+  refund_amount,
+  refund_reason,
+  products,
+});
+```
 
-setPageType({ type, language, locale });
+- Where `transaction_id` is the ID of the transaction to be refunded.
+- Where `currency` is the currency used for the transaction.
+- Where `refund_amount` is the monetary amount that was refunded.
+- Where `refund_reason` is the reason the refund took place.
+- Where `products` is the array of products which were refunded on the transaction. Refunds might be partial, so only the products that were refunded should be sent.
+
+**Example usage:**
+
+```ts
+import { trackRefund } from "@snowplow/browser-plugin-snowplow-ecommerce";
+
+trackRefund({
+  transaction_id: "T12345",
+  currency: "USD",
+  refund_amount: 200,
+  refund_reason: "false_size",
+  products: [
+    {
+      id: "P125",
+      name: "Baseball T",
+      brand: "Snowplow",
+      category: "Mens/Apparel",
+      price: 200,
+      currency: "USD",
+    },
+  ],
+});
 ```
 
 {{% /tab %}}
 {{% tab name="JavaScript API" %}}
 
+#### `trackRefund`
+
+To track a refund you can use the `trackRefund` method with the following attributes:
 
 ```ts
 /* {trackerName} is a placeholder for the initialized tracker on your page.  */
 
-window.snowplow("setPageType:{trackerName}", { type, language, locale });
+window.snowplow("trackRefund:{trackerName}", {
+  transaction_id,
+  currency,
+  refund_amount,
+  refund_reason,
+  products,
+});
 ```
 
-{{% /tab %}}
+- Where `transaction_id` is the ID of the transaction to be refunded.
+- Where `currency` is the currency used for the transaction.
+- Where `refund_amount` is the monetary amount that was refunded.
+- Where `refund_reason` is the reason the refund took place.
+- Where `products` is the array of products which were refunded on the transaction. Refunds might be partial, so only the products that were refunded should be sent.
 
-{{< /tabs >}}
+**Example usage:**
 
-- Where `type` is the type of page that was visited. E.g. homepage, product page, cart.
-- Where `language` is the language the web page is based in.
-- Where `locale` is the locale version of the site that is running.
+```ts
+window.snowplow("trackRefund:{trackerName}", {
+  transaction_id: "T12345",
+  currency: "USD",
+  refund_amount: 200,
+  refund_reason: "false_size",
+  products: [
+    {
+      id: "P125",
+      name: "Baseball T",
+      brand: "Snowplow",
+      category: "Mens/Apparel",
+      price: 200,
+      currency: "USD",
+    },
+  ],
+});
+```
 
-#### `setEcommerceUser`
+{{% /tab %}}
 
-To set an Ecommerce User context you can use the `setEcommerceUser` method with the following attributes:
+{{% tab name="Node.js" %}}
```

Comment:
Are you planning on adding Node.js tabs for the other tracking guides too?

Reply:
Nope, this is the only place for now _(could probably add this on the transaction error as well)_ since it is the one interaction which probably will happen mostly on the backend.